src/components/profile/static/ProfileDetails.js

```diff
@@ -0,0 +1,69 @@
+import React, { Fragment } from 'react';
+import dayjs from 'dayjs';
+import { Link } from 'react-router-dom';
+
+// MUI
+import MuiLink from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+
+// Icons
+import CalendarToday from '@material-ui/icons/CalendarToday';
+import LinkIcon from '@material-ui/icons/Link';
+import LocationOn from '@material-ui/icons/LocationOn';
+
+const ProfileDetails = ({ handle, bio, createdAt }) => {
+  const createBio = () => {
+    if (!!bio.aboutMe) {
+      return <Typography variant='body2'>{bio.aboutMe}</Typography>;
+    }
+    return <div />;
+  };
+
+  const createLocation = () => {
+    if (!!bio.location) {
+      return (
+        <Fragment>
+          <LocationOn color='primary' />
+          <span>{bio.location}</span>
+          <hr />
+        </Fragment>
+      );
+    }
+    return <div />;
+  };
+
+  const createWebsite = () => {
+    if (bio.website) {
+      const link = (
+        <a href={bio.website} target='_blank' rel='noopener noreferrer'>
+          {' '}
+          {bio.website}
+        </a>
+      );
+      return (
+        <Fragment>
+          <LinkIcon color='primary' />
+          {link}
+          <hr />
+        </Fragment>
+      );
+    }
+    return <div />;
+  };
+  return (
+    <div className='profile-details'>
+      <MuiLink component={Link} to={`/${handle}`} color='primary' variant='h5'>
+        @{handle}
+      </MuiLink>
+      <hr />
+      {createBio()}
+      <hr />
+      {createLocation()}
+      {createWebsite()}
```

Comment:
If these aren't doing tons of logic, I've been starting to lean towards treating them as components, so you have:
```
<UserLocation />
<UserWebsite />
```

Reply:
It was originally done because it was causing too many indentations, but you might be right about making them their own components. I'll be pushing those out soon.

Reply:
Fixed